drv/lpc55-spi/src/lib.rs

```diff
@@ -225,13 +255,74 @@ impl Spi {
         self.reg.fifointstat.read().bits()
     }
 
+    // 35.6.8 SPI interrupt status register
+    // Reading this register clears the interrupt conditions.
+    //
+    // Slave select assert - set on transitions from de-asserted to asserted.
+    // Slave select de-assert - set on transitions from asserted to de-asserted.
+    // Master idle status flag - true when master function is fully idle.
+    pub fn intstat(&self) -> (bool, bool, bool) {
+        let stat = self.reg.intstat.read();
+        (stat.ssa().bits(), stat.ssd().bits(), stat.mstidle().bits())
+    }
+
+    pub fn stat(
+        &mut self,
+    ) -> (bool, bool, bool, bool, bool, bool, bool, u8, u8) {
+        let stat = self.reg.fifostat.read();
```

Comment:
It's hard to follow all the values of the returned tuple here and other places. Can you just return the value of `fifostat.read()` and call the appropriate function instead?